src/jit/codegenarm64.cpp

```diff
@@ -3183,15 +3220,28 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
     case GT_PUTARG_REG:
         if (targetType == TYP_STRUCT)
```

Comment:
Here I think we DO need varTypeIsStruct, unless all non-TYP_STRUCT structs are passed in a single register (which I don't think is the case here).